baked_in.go

```diff
@@ -304,12 +304,12 @@ func isSSN(fl FieldLevel) bool {
 
 // IsLongitude is the validation function for validating if the field's value is a valid longitude coordinate.
 func isLongitude(fl FieldLevel) bool {
-	return longitudeRegex.MatchString(fl.Field().String())
+	return longitudeRegex.MatchString(fmt.Sprint(fl.Field().Interface()))
```

Comment:
like the change, it would be nice if you could add a type switch as it will allow the conversion from int to string to be much more efficient eg. https://github.com/go-playground/validator/blob/52ea44899849049e3ae9267ef1902c36807fe198/baked_in.go#L193

Reply:
Did a small benchmark to compare and you're right. Check the latest commits.